app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -43,6 +43,8 @@
     <icon drawable="@drawable/app_manager" package="cz.seeq.prog.android.packageviewer" name="Package Viewer" />
     <icon drawable="@drawable/app_manager" package="io.github.muntashirakon.AppManager" name="App Manager" />
     <icon drawable="@drawable/app_manager_debug" package="io.github.muntashirakon.AppManager.debug" name="App Manager Debug" />
+    <icon drawable="@drawable/airguard" package="de.seemoo.at_tracking_detection" name="AirGuard" />
+    <icon drawable="@drawable/airguard" package="de.seemoo.at_tracking_detection.release" name="AirGuard" />
```

Comment:
Need to resort like d33eb8e9006cfbf61aea33b93a01bb5e41480a03.

Reply:
Resolved in https://github.com/LawnchairLauncher/lawnicons/commit/d33eb8e9006cfbf61aea33b93a01bb5e41480a03.